sopel/modules/admin.py

```diff
@@ -44,6 +44,27 @@ def setup(bot):
     bot.config.define_section('admin', AdminSection)
 
 
+class InvalidSection(Exception):
+    def __init__(self, section):
+        super(InvalidSection, self).__init__(self)
```

Comment:
No need for a `@property` on `message` for that:

```suggestion
        super(InvalidSection, self).__init__(self, 'Section [{}] does not exist.'.format(section))
```

Reply:
Done.